scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -417,7 +416,11 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                                     .split(',');
 
                                 for (var i = 0; i < modifiedErrors.length; i++) {
-                                    notify.error(_.trim(modifiedErrors[i]));
+                                    var message = _.trim(modifiedErrors[i]);
+                                    var field = helpers.getFirstWord(message).substr(1);
```

Comment:
You can just use `message.split(" ")[0]` instead of creating this new method `getFirstWord` which does the same thing.